model/be-nc-news.model.js

```diff
@@ -1,17 +1,23 @@
-const db = require("../db/connection")
-const data = require("../db/data/test-data/index")
+const db = require("../db/connection");
+const data = require("../db/data/test-data/index");
 
 // require("jest-sorted")
 
 function selectTopics() {
-    return db.query
-    ("SELECT * FROM topics").then((result) => {
-        return result.rows
-    })
+  return db.query("SELECT * FROM topics").then((result) => {
+    return result.rows;
+  });
 }
 
-
+function selectArticlesById(articleId) {
+  return db
+    .query(`SELECT * FROM articles WHERE article_Id = $1`, [articleId])
+    .then((result) => {
+      return result.rows[0];
```

Comment:
So in here, there was no article with id 9999, an empty array would be returned - if this was the case we can force this to reject with Promise.reject({msg: some error}) - this would then get passed onto the catch block of our controller which would then pass it on to your error handling middleware. 

If you get stuck on this, I would suggest dropping an nchelp